packages/react/src/index.ts

```diff
@@ -11,3 +14,7 @@ export { useSchemaDocumenationStore } from './schema-documentation/index';
 // schema-dependent components
 export { SchemaDocumentation } from './schema-documentation';
 export { SchemaView } from './schema-view';
+
+if (import.meta.__IS_LITE_MODE_ !== 'true') {
+  setMonacoImporter(() => import('monaco-graphql/esm/monaco-editor'));
```

Comment:
This new import strategy is breaking the ladle setup (`pnpm ladle:dev`). Is there an alternative here that doesn't affect ladle?

Reply:
I've pushed an update that fixes this.